custom-recipes/amazon-transcribe/recipe.py

```diff
@@ -1,45 +1,46 @@
 # -*- coding: utf-8 -*-
+import uuid
 from amazon_transcribe_api_client import AWSTranscribeAPIWrapper
 from dku_io_utils import read_json_from_folder, set_column_description
 from dkulib.core.parallelizer import DataFrameParallelizer
 from plugin_params_loader import PluginParamsLoader
 from plugin_params_loader import RecipeID
 
+
 # ==============================================================================
-# CONSTANT DEFINITION
+# CONSTANT/PATTERN DEFINITION
 # ==============================================================================
 
-RECIPE_JOB_ID = dataiku.dku_custom_variables.get('jobId')
+RECIPE_JOB_ID_PATTERN = f"{dataiku.dku_custom_variables.get('jobId')}_{uuid.uuid4().hex}"
```

Comment:
nitpick, I wouldn't call it a pattern since it's an actual value

Reply:
done in 1b4d2379271f3e34715ff74b42eb6f17053e741a